Sources/HiPMobile/Helpers/Settings.cs

```diff
@@ -254,7 +254,7 @@ public static string UserId
         /// </summary>
         private const string UserNameKey  = "username_key";
 
-        private static readonly string UsernameDefault = "abc"; //proper username instead of max@power.com
+        private static readonly string UsernameDefault = "hipadmin"; //proper username instead of max@power.com
```

Comment:
Why is this `hipadmin` and not an empty string?

Reply:
If we keep it as an empty string then we do not have any username for our default login. We can keep it as an empty string as well, what do you suggest @cbruegg ?

Reply:
If this app was to ever face users, then they shouldn't see something like `hipadmin` in the UI, because they will never use this username to log in. So I think it would make sense to change this to an empty string, do you agree? I may also be getting something wrong.

Reply:
i think hipadmin is better, since otherwise nothing will be displayed here.

Reply:
Wouldn't a regular user be confused to see this username? Or maybe I'm confused, where exactly will this default value be shown in the app? Or won't it be shown?

Reply:
It will only be shown when a user logins with debug mode and will be displayed on the profile page.

Reply:
In that case, I think this is okay to merge